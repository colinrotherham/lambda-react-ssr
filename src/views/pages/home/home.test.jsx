import React from 'react'
import renderer from 'react-test-renderer'
import Home from './home'

describe('Page: Home', () => {
  test('Component is rendered', () => {
    const component = renderer.create(<Home />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
