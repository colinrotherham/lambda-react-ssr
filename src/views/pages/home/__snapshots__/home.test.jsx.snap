// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page: Home Component is rendered 1`] = `
<html
  lang="en-GB"
>
  <head>
    <meta
      charSet="utf-8"
    />
    <title>
      Hello World
    </title>
    <meta
      content="width=device-width, initial-scale=1, viewport-fit=cover"
      name="viewport"
    />
  </head>
  <body>
    <header
      className="header"
    >
      <h1>
        Hello header
      </h1>
      <p>
        Yes this is a header
      </p>
    </header>
    <footer
      className="footer"
    >
      <h2>
        Hello footer
      </h2>
      <p>
        Yes this is a footer
      </p>
    </footer>
  </body>
</html>
`;
